hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -41,10 +41,7 @@
 import java.util.Optional;
 import java.util.Set;
 import java.util.Objects;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.LinkedBlockingQueue;
-import java.util.concurrent.ThreadPoolExecutor;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;
```

Comment:
Wildcard should be avoided.

Reply:
oops will do - I forgot to set this up on my new laptop